spec/datetimeformat.html

```diff
@@ -144,29 +136,26 @@
       </p>
 
       <emu-alg aoid="ToDateTimeOptions">
-        1. If _options_ is *undefined*, then let _options_ be *null*, else let _options_ be ToObject(_options_).
-        1. ReturnIfAbrupt(_options_).
+        1. If _options_ is *undefined*, let _options_ be *null*; otherwise let _options_ be ? ToObject(_options_).
         1. Let _options_ be ObjectCreate(_options_).
         1. Let _needDefaults_ be *true*.
         1. If _required_ is *"date"* or *"any"*,
           1. For each of the property names *"weekday"*, *"year"*, *"month"*, *"day"*:
             1. Let _prop_ be the property name.
-            1. Let _value_ be Get(_options_, _prop_).
-            1. ReturnIfAbrupt(_value_).
-            1. If _value_ is not *undefined*, then let _needDefaults_ be *false*.
+            1. Let _value_ be ? Get(_options_, _prop_).
+            1. If _value_ is not *undefined*, let _needDefaults_ be *false*.
         1. If _required_ is *"time"* or *"any"*,
           1. For each of the property names *"hour"*, *"minute"*, *"second"*:
             1. Let _prop_ be the property name.
-            1. Let _value_ be Get(_options_, _prop_).
-            1. ReturnIfAbrupt(_value_).
-            1. If _value_ is not *undefined*, then let _needDefaults_ be *false*.
+            1. Let _value_ be ? Get(_options_, _prop_).
+            1. If _value_ is not *undefined*, let _needDefaults_ be *false*.
         1. If _needDefaults_ is *true* and _defaults_ is either *"date"* or *"all"*, then
           1. For each of the property names *"year"*, *"month"*, *"day"*:
-            1. Let _status_ be CreateDatePropertyOrThrow(_options_, _prop_, *"numeric"*).
+            1. Let _status_ be the result of calling CreateDatePropertyOrThrow(_options_, _prop_, *"numeric"*).
```

Comment:
There really isn't a difference between `let _f_ be Call()` and `Let _f_ be the result of calling`. The latter is an outmoded more verbose convention that can be replaced with the former any time its spotted. You are correct that there's an oddity with completion records - the oddity is that sometimes algorithms treat a completion record as if it were a value. This is ok per 6.2.2.2 last paragraph.

I completely agree that depending on the 6.2.2.2 implicitness is bad and now that it's easy to be explicit, we should be. Proposals for making this easier are welcome! (Eg. I had originally proposed ! as a short-hand for asserting an operation is not an abrupt completion and replacing the completion record with its value, for example).